lib/themes/dosomething/paraneue_dosomething/js/components/ContextSlide.js

```diff
@@ -19,7 +19,7 @@ class ContextSlide extends React.Component {
       campaign: {
         title: 'Pride Over Predjudice',
         tagline: 'Post a selfie to combat anti-immigrant hate speech online.',
-        image: 'http://placekitten.com/352/337',
+        image: 'http://placekitten.com/352/264',
```

Comment:
We should probably specify these dimensions in CSS at some point if it has to be exactly this